wfl/DBManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Configuration;
+using System.Threading.Tasks;
+using MongoDB.Driver;
+using wfl.Models;
+
+namespace wfl
+{
+    public class DBManager
+    {
+        private static string connectionString;
+        public static MongoClient client;
+        public static IMongoDatabase db;
+
+        static DBManager()
+        {
+            connectionString = ConfigurationManager.ConnectionStrings["databaseConnectionString"].ConnectionString;
```

Comment:
Seems like either will work in this case. Although, since WebConfigurationManager is preferred for ASP.NET, I agree it might be a good idea to use WebConfigurationManager.